scc/light_client/provider/provider.go

```diff
@@ -0,0 +1,20 @@
+package provider
+
+import (
+	"github.com/0xsoniclabs/sonic/scc"
+	"github.com/0xsoniclabs/sonic/scc/cert"
+	"github.com/Fantom-foundation/lachesis-base/inter/idx"
+)
+
+//go:generate mockgen -source=provider.go -package=provider -destination=provider_mock.go
+
+// Provider is an interface to access certificates of the Sonic Certification Chain.
+type Provider interface {
+	// GetCommitteeCertificates returns the committee certificates for the
+	// period speciefied starting from first and returning up to 'maxResults' entries.
```

Comment:
The grammar is a bit off in this documentation ... maybe replace with
```suggestion
	// GetCommitteeCertificates returns up to `maxResults` consecutive committee 
	// certificates starting from the given period. 
```

Reply:
applied in commitee as well as in block certificates documentation.